resources/lang/en/identities.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+return [
+
+    'hint' => 'These are the third party services you connected or used for sign in or sign up on the K-Box. From this page you can manage the connection and see how we use your information.',
+
+    'providers' => [
+
+    ],
+
+    'connected_at' => 'Connected on',
+    
+    'registration' => 'Used for sign-up',
+
+    'unlink' => 'Unlink',
+    'unlinking' => 'Unlinking...',
+
+    'delete' => [
+        'question' => 'Unlink :Provider?',
+
+        'message' => 'You will lose the ability to login via :Provider and use the offered integration. Existing data will be kept.',
+        'registration_message' => 'This identity was used for sign-up.',
+        'registration_with' => 'The account was registered with the email',
+        'registration_set_password' => 'Setup a password not to lose access',
```

Comment:
@avvertix maybe we present it as "Setup a password to preserve access"?

Reply:
Yes, fine for me